### PR TITLE
chore: add golang

### PR DESCRIPTION
**Pull request by thinkingserious**
To see how the web page looks: https://refined-github-html-preview.kidonng.workers.dev/twilio/twilio.github.io/raw/golang/index.html

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by JenniferMah — COMMENTED** [review]
Can you also update the [auto-generated documentation wiki](https://wiki.hq.twilio.com/display/DI/Auto-Generated+Documentation) with the steps you took to update the s3?

**Comment by thinkingserious** [comment]
Yes. Before deploying to s3, I need approval on this PR. The steps to deploy are currently documented in the working doc, but will be added to our wiki once verified.

**Review by JenniferMah — APPROVED** [review]

**Pull request merged**
